### DISCUSSION TITLE
Hindi Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#363] [Language Leaders]
Discussion to elect language leader(s) for Hindi.

**Comment by arunshekher** [comment]
May I know why there is no discussion categories for any of the Indo-Dravidian languages?

**Reply by nagyrobi** [reply]
Please suggest a category name and a flag icon.

**Reply by synesthesiam** [reply]
Github has a limit of 25 categories in a repo for some ridiculous reason, so I'm having to move everything to labels.

**Reply by arunshekher** [reply]
@nagyrobi For Indo-Dravidian family?  I'd suggest 4 categories/labels for these family of languages. Flag icon :  🇮🇳
1. `Tamil`
2. `Telugu`
3. `Kannada`
4. `Malayalam`

**Comment by synesthesiam** [comment]
@arunshekher some discussions here:

* Kannada: https://github.com/home-assistant/intents/discussions/431
* Telugu: https://github.com/home-assistant/intents/discussions/419

**Reply by arunshekher** [reply]
@synesthesiam thank you for the links. I went through those discussions already. My native tongue is Malayalam (മലയാളം) and I am looking forward to contributing to that language. Though I think I know enough of syntax and grammar to contribute to Hindi as well but I only learned it as a second language for about 7 years in school.
